app/recipeApp.js

```diff
@@ -0,0 +1,339 @@
+let recipeObject
+
+let recipes = []; 
+
+async function getData() {
```

Comment:
consider adding a try/catch to avoid hard failure (exceptions) when when fetch returns an error or when you fail to read json

Reply:
See https://dmitripavlutin.com/javascript-fetch-async-await/